packages/core/src/app.tsx

```diff
@@ -16,68 +18,67 @@ import { BasicExample as ConfirmNavigationExample } from '@servicetitan/confirm-
-                                    Table (Master/Detail)
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/table-state-caching">
-                                    Table (State Caching)
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/confirm" exact>
-                                    Confirm
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/confirm-navigation">
-                                    Confirm Navigation
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/file-uploader">
-                                    File Uploader
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/number-input">
-                                    Number Input
-                                </SideNavLinkItem>
-                                <SideNavLinkItem pathname="/new/examples/notifications">
-                                    Notifications
-                                </SideNavLinkItem>
-                            </SideNav>
-                        </Sidebar.Section>
-                    </Sidebar>
-                }
-                spacing="relaxed"
-                fullHeight
-            >
-                <Switch>
-                    <Route path="/new/examples/table" component={TableExample} />
-                    <Route
-                        path="/new/examples/table-master-detail"
-                        component={TableMasterDetailExample}
-                    />
-                    <Route
-                        path="/new/examples/table-state-caching"
-                        component={TableStateCachingExample}
-                    />
-                    <Route path="/new/examples/confirm" component={ConfirmExample} />
-                    <Route
-                        path="/new/examples/confirm-navigation"
-                        component={ConfirmNavigationExample}
-                    />
-                    <Route path="/new/examples/file-uploader" component={FileUploaderExample} />
-                    <Route path="/new/examples/number-input" component={NumberInputExample} />
-                    <Route path="/new/examples/notifications" component={NotificationsExample} />
-                    <Redirect path="/new/examples/" to="/new/examples/table" />
-                </Switch>
-            </Page>
-        </HashBrowserRouter>
-    </React.StrictMode>
-);
+export const App: React.FC = () => {
+    const [basename = '/new/examples'] = useOptionalDependencies(BASENAME_TOKEN);
+
+    return (
+        <React.StrictMode>
+            <HashBrowserRouter>
```

Comment:
Don't we support basename as a prop for HashBrowserRouter?

Reply:
It adds it to the wrong place:
![Screenshot 2021-02-17 at 11 48 49](https://user-images.githubusercontent.com/17852627/108180959-65712600-7118-11eb-81df-eb6c19802901.png)

Reply:
I guess we can try to rework `HashBrowserRouter`.